api/rs/slint/Cargo.toml

```diff
@@ -153,6 +153,7 @@ backend-linuxkms = ["i-slint-backend-selector/backend-linuxkms", "std"]
 i-slint-core = { version = "=1.2.0", path = "../../../internal/core", default-features = false }
 slint-macros = { version = "=1.2.0", path = "../macros" }
 i-slint-backend-selector = { version = "=1.2.0", path = "../../../internal/backends/selector" }
+i-slint-renderer-femtovg = { version = "=1.2.0", path = "../../../internal/renderers/femtovg", optional = true, default-features = false }
```

Comment:
there are no default features on i-slint-renderer-femtovg

Reply:
Yeah, I just wanted to be on the safe side for the future :). After all here we don’t care about the features, merely availability.